includes/sanitizers/class-amp-block-sanitizer.php

```diff
@@ -0,0 +1,131 @@
+<?php
+/**
+ * Class AMP_Block_Sanitizer.
+ *
+ * @package AMP
+ */
+
+/**
+ * Class AMP_Block_Sanitizer
+ *
+ * Modifies elements created as blocks to match the blocks' AMP-specific configuration.
+ */
+class AMP_Block_Sanitizer extends AMP_Base_Sanitizer {
```

Comment:
@westonruter That's still WIP but if you happen to have time, could you check if this approach for embeds seems to make sense?

Reply:
I think you're on the right track!